simulator/simple_planning_simulator/test/test_simple_planning_simulator.cpp

```diff
@@ -263,13 +260,9 @@ TEST_P(TestSimplePlanningSimulator, TestIdealSteerVel)
   rclcpp::shutdown();
 }
 
-const std::string VEHICLE_MODEL_LIST[] = {
-  "IDEAL_STEER_VEL",
-  "IDEAL_STEER_ACC",
-  "IDEAL_STEER_ACC_GEARED",
-  "DELAY_STEER_VEL",
-  "DELAY_STEER_ACC",
-  "DELAY_STEER_ACC_GEARED",
+const char VEHICLE_MODEL_LIST[] = {
+  "IDEAL_STEER_VEL", "IDEAL_STEER_ACC", "IDEAL_STEER_ACC_GEARED",
+  "DELAY_STEER_VEL", "DELAY_STEER_ACC", "DELAY_STEER_ACC_GEARED",
 };
 
 INSTANTIATE_TEST_CASE_P(
```

Comment:
I do not know why but this code no longer builds.
The following changes allow the build to succeed.
```suggestion
const std::string VEHICLE_MODEL_LIST[] = {
  "IDEAL_STEER_VEL", "IDEAL_STEER_ACC", "IDEAL_STEER_ACC_GEARED",
  "DELAY_STEER_VEL", "DELAY_STEER_ACC", "DELAY_STEER_ACC_GEARED",
};

INSTANTIATE_TEST_SUITE_P(
```

Reply:
Thank you.
Hmm, I've changed here following the `cpplint`'s advice.
I'll revert this change and add `// NOLINT`.

Reply:
And I guess `INSTANTIATE_TEST_CASE_P` has no problem.